lab_1_keywords_tfidf/main.py

```diff
@@ -4,20 +4,31 @@
 """
 from typing import Optional, Union
 
+from lab_1_keywords_tfidf.assets.freq_and_idf_dictionary_creation import file
+
+
+def clean_and_tokenize(text: str) -> [list[str]]:
```

Comment:
Students  are not supposed to change the line with definition of the function
Please, keep it `def clean_and_tokenize(text: str) -> Optional[list[str]]:`

Reply:
okey